### PR TITLE
chat page template

### PR DESCRIPTION
**Pull request by naphatbass**

### CODE REVIEW

File: src/pages/user/chat.tsx [hunk 1]
```diff
@@ -1,9 +1,105 @@
...
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleChatBoxClick = () => {
+    setIsClicked(!isClicked);
+  };
+
   return (
-    <div className="min-h-screen w-full h-full flex items-center justify-center text-3xl">
-      Coming soon....
+    <div
```
Comment (noppakorn) [inline src/pages/user/chat.tsx:13]: more margin and padding? I think each chat box is too close to each other.

**Review by noppakorn — CHANGES_REQUESTED** [review]

File: src/pages/user/chat.tsx [hunk 2]
```diff
@@ -1,9 +1,105 @@
...
+          </button>
+        </div>
+        <div className="h-full w-full overflow-y-scroll">
+          <ChatBox name="Lalisa Manobal" message="last message"/>
+          <ChatBox name="Jennie Kim" message="last message"/>
+          <ChatBox name="Jisoo Kim" message="last message"/>
+          <ChatBox name="Chaeyong Pak" message="last message"/>
+        </div>
+      </div>
+      {showModal && (
```
Comment (nnanthicha) [inline src/pages/user/chat.tsx:81]: I just create component template for modal, use that one instead.

**Review by nnanthicha — CHANGES_REQUESTED** [review]
Also fix margin or padding

File: src/pages/user/chat.tsx [hunk 3]
```diff
@@ -1,9 +1,105 @@
-const Chat = () => {
+import { useState } from "react";
+import { PlusIcon, XMarkIcon} from "@heroicons/react/24/outline";
+import Image from "next/image";
+
+const ChatBox = (props:{name:string,message:string}) => {
+  const [isClicked, setIsClicked] = useState(false);
```
Comment (nnanthicha) [inline src/pages/user/chat.tsx:6]: click behavioral is not correct, it should be gray only one box at a time.

**Review by noppakorn — APPROVED** [review]
Wait for @nnanthicha review

**Review by nnanthicha — APPROVED** [review]

**Pull request merged**